etc/generated_headers/mongoc-config.h

```diff
@@ -48,7 +43,7 @@
 #endif
 
 #if MONGOC_ENABLE_SSL_OPENSSL && OPENSSL_VERSION_NUMBER < 0x10100000L
-#  define MONGOC_HAVE_ASN1_STRING_GET0_DATA 0
+// #  define MONGOC_HAVE_ASN1_STRING_GET0_DATA 0
```

Comment:
The value is defined above in the `__APPLE__` ifdef, it's always set so this would generate a compiler warning for setting it twice.

Reply:
ah I see, GitHub collapsed that part above so I missed it.

Reply:
Can we just delete this section then?

Reply:
sure

Reply:
I don't really feel strongly about it, but it just seems like extra clutter to me. We don't have plans on uncommenting this portion ever, right? And the resultant diff would just be this section being deleted entirely as opposed to a comment addition, right?

Reply:
ok, removed

Reply:
Now that I'm looking at the travis failures on Linux, I think we actually do need the original uncommented version. Maybe also add an `&& !defined(__APPLE__)` to it to avoid the warnings? Sorry, I should have checked this before commenting about removing it.